ovos_plugin_manager/templates/tts.py

```
@@ -262,7 +262,10 @@ def preprocess_sentence(self, sentence: str) -> List[str]:
             list: list of sentence parts
         """
         if self.config.get("sentence_tokenize"):  # TODO default to True on next major release
-            return quebra_frases.sentence_tokenize(sentence)
+            try:
+                return quebra_frases.sentence_tokenize(sentence)
+            except:
```

Ruff (0.15.4)

[error] Do not use bare `except`

(E722)